danul-code/new.html

```diff
@@ -0,0 +1,102 @@
+            <textarea id="body" name="body" type="text"></textarea>
+          <label for="author">Article Author:</label>
+            <input id="author" name="author" type="text" />
+          <label for="url">Author URL:</label>
+            <input id="url" name="url" type="text" />
+          <label for="cat">Article Category:</label>
+            <input id="cat" name="cat" type="text" />
+          <label for="published">Published:</label>
+            <input id="published" name="published" type="checkbox" />
+        </form>
+        <!-- DONE: Set up a form to accept user input for a new blog article. Remember that we need:
+           1. Article title
+           2. Article body
+           3. Article author
+           4. Author's URL
+           5. Article category
+           6. Some way to know if this article is a draft or is published -->
+      </section>
+
+      <section id="articles" class="tab-content">
+        <!-- NOTE: This is where we will display our article preview -->
+      </section>
+
+      <section id="article-export">
+        <!-- TODO: Refer to the preview image to see what we will be building here: we need a container of some kind to hold a JSON export of our article. Ideally this container would show only a incipit (short preview) of the JSON and also be clickable to allow easy selection of the text it holds. What kind of element could we use for this? -->
+        <input type="text" disabled id="export" />
+      </section>
+
+    </main>
+    <footer>
+      &copy; 2018 Code Fellows | Static Blog | Happy Ipsum!
+    </footer>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.11/handlebars.min.js"></script>
+    <!-- NOTE (stretch goal): Include script tags for 3rd party libraries to enable Markdown rendering and code syntax highlighting. Use the minified CDN. Keep in mind that if we want these features to render properly on our main blog page (index.html) that we'll need to add those scripts over there, too. -->
+    <script src="scripts/blogArticles.js"></script>
+    <script src="scripts/article.js"></script>
+    <script src="scripts/articleView.js"></script>
+    <script>
+      // DONE: We will need to call a function here to initialize the behaviors on this page, once we get them built out. Why?
+      // (Look at the bottom of index.html as a reference. Since we are now using our JS on two different HTML pages that have different needs as to what parts of the code they utilize, we need to build a function that inializes each page.)
+      //  Hint: Look in articleView.js for a function that does this.
+      articleView.initNewArticlePage();
+      // COMMENT: What is this function doing? Why do we need it?
+      // This function is loading the functions required for only this page.  This keeps from unnecessary files from loading for this page.
```

Comment:
actually, this function is making the form elements dynamic so that when you enter input, the preview shows and you are able to work with your new blog article.